src/modules/data/commands.js

```diff
@@ -33,7 +38,25 @@ const mkdir = new Command(
 const echo = new Command(
     'echo - Write arguments to the standard output.',
     '',
-    function echo() {}
+    function echo(argument, parameter = []) {
+        if (parameter.length > 0) {
+            argument.forEach(e => {
+                if (e === '>') {
+                    let indexOfBiggerThan = argument.indexOf(e);
```

Comment:
Este foreach te lo puedes ahorrar por lo que veo, no? Bastaría que comprobaras si indexOf('>') devuelve un valor y si lo devuelve ya tienes justo el índice que es lo que estás buscando

Reply:
Si ademas estoy mirando si el elemento esta en argumentos que no es asi, al final ahi he puesto si el parametro es igual a > que es lo que tiene realmente sentido. El for each era para reagrupar la string que se guarda en el archivo cosa que seguro puedo hacer mas facil.

Reply:
Vale ya lo he dejado mejor, ahora se echo puede escribir la misma linea en varios archivos. 

```bash
echo this is a line of text > file1 file2 file3
```
Me faltaria crear los archivos en el path actual que aun no he mirado como se puede hacer. 
Y el codigo tiene mucho mas sentido ahora.

Reply:
Tambien que cuando el archivo ya exista sobrescriba lo que hay dentro con la string.